skimage/feature/tests/test_haar_new.py

```diff
@@ -0,0 +1,232 @@
+# File name: test_haar_new.py
+# Contributor: Amazon Lab126 Multimedia team
```

Comment:
Are these tests compatible with the old haar features? If so, maybe it makes sense to have them in a seperate PR so as to enable a distinc review for these additions?